apps/backend/src/location/entities/location.entity.ts

```diff
@@ -10,11 +10,13 @@ import { BankLocationEntity, MerchantEntity } from '.';
 export class MinistryLocationEntity extends BaseLocationEntity {
   @OneToMany(() => BankLocationEntity, (bank) => bank.location, {
     cascade: true,
+    eager: true,
```

Comment:
Please justify these eagers

Reply:
We almost always need these together - at least to integrate with the code easily. Could be removed later if we refactor.

Reply:
I removed the eager from pos deposit and cash deposit, in the case of locations though we need the banks/merchants in most cases

Reply:
...not sure if my response is sufficient - please let me know if there is any action needed to resolve 😁